compiler/gcc-cross-compile.cmake

```diff
@@ -0,0 +1,50 @@
+# Copyright (c) 2015 Alexandre Pretyman
+# All rights reserved.
+#
+# ------------------------------------------------------------------------------
+#  GCC Based Cross Compiler CMake toolchain file
+#
+#  Variables:
+#   CROSS_COMPILE_TOOLCHAIN_PATH=/path/to/toolchain/bin
+#   CROSS_COMPILE_TOOLCHAIN_PREFIX=arm-unknown-linux-gnueabihf
+#   CROSS_COMPILE_SYSROOT=/path/to/sysroot [optional]
```

Comment:
I guess this is a CMake variables that must be set in toolchain file from the top. Probably we need to check they are not empty and report descriptive error message otherwise.